File: app/src/main/java/com/tt/handsomeman/di/module/SignUpAddPayoutModule.java
package com.tt.handsomeman.di.module;

import com.tt.handsomeman.service.SignUpAddPayoutService;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;
import retrofit2.Retrofit;

@Module
public class SignUpAddPayoutModule {
    @Provides
    @Singleton
    SignUpAddPayoutService provideSignUpAddPayoutService(Retrofit retrofit) {
        return retrofit.create(SignUpAddPayoutService.class);
    }
}

File: app/src/main/java/com/tt/handsomeman/di/module/SignUpAddPayoutModule.kt
package com.tt.handsomeman.di.module

import com.tt.handsomeman.service.SignUpAddPayoutService

import javax.inject.Singleton

import dagger.Module
import dagger.Provides
import retrofit2.Retrofit

@Module
class SignUpAddPayoutModule {
    @Provides
    @Singleton
    internal fun provideSignUpAddPayoutService(retrofit: Retrofit): SignUpAddPayoutService {
        return retrofit.create(SignUpAddPayoutService::class.java!!)
    }
}
